install/tools/ipa-managed-entries

```diff
@@ -72,7 +73,7 @@ def main():
         sys.exit("Unrecognized action [" + args[0] + "]")
     standard_logging_setup(None, debug=options.debug)
 
-    api.bootstrap(context='cli', debug=options.debug)
+    api.bootstrap(context='cli', debug=options.debug, confdir=paths.ETC_IPA)
```

Comment:
Nvm, the script behaves the same in other error cases, disregard the previous comment.